### DISCUSSION TITLE
[WASM] Build error MSB4186: Invalid static method invocation syntax: "[System.String]::Copy().Contains('Microsoft.NET.Sdk.Web')"

### DISCUSSION THREAD

**Discussion by tobizk04** [unoplatform/uno#16851] [Q&A] [answered]
Hello.
I have updated to the new Uno platform version and now get the following error on build:

-> C:\Users\xxxxxx\.nuget\packages\uno.sdk\5.2.121\targets\Uno.Common.Wasm.targets(60,43): error MSB4186: Invalid static method invocation syntax: "[System.String]::Copy().Contains('Microsoft.NET.Sdk.Web')". Method 'System.String.Copy' not found. Static method invocation should be of the form: $([FullTypeName]::Method()), e.g. $([System.IO.Path]::Combine(`a`, `b`)). 
Check that all parameters are defined, are of the correct type, and are specified in the right order.

All my colleagues have the same problem.
It's not a problem on one machine.

How could I fix that?

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. This was fixed in a later version of uno platform. You can find how to [update the uno.sdk here](https://platform.uno/docs/articles/upgrading-nuget-packages.html). 5.2.139 was released a few hours ago, give it a try and let us know if it helped!

**Comment by tobizk04** [comment]
@jeromelaban
Thanks for your fast answer.
The sdk version 5.2.139 fix the bug. It works!
Have a nice weekend!
